recipes/python-symengine/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.1.0.git" %}
+
```

Comment:
You can add the commit as a jinja variable here:

`{% set commit = "4361e3782ce2b61c9962f3f9a1e766e481a00bf9" %}`